third_party/move/move-compiler-v2/src/pipeline/abort_analysis.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+//! Computes at a given program point, how the program may exit later.
+//! Does the program returns, aborts, or doesn't terminate?
+//! Check documentation of `ExitState` for more on the abstract domain used in the analysis.
+//! The analysis is intraprocedural, and considers any user function may abort.
+//!
+//! Requires: The program cannot silently exits, i.e., neither returns nor aborts, but runs out of instructions.
```

Comment:
Are you sure that can't happen?  It would be nice if we could describe such stackless bytecode invariants/assumptions somewhere.
@wrwg @vineethk

Reply:
tests added for calling user functions.

Reply:
@fEst1ck Can you please confirm if this is resolved?